test/unit/lib/plugins/aws/invoke.test.js

```diff
@@ -13,7 +13,7 @@ chai.use(require('sinon-chai'));
 
 const expect = chai.expect;
 
-describe('test/unit/lib/plugins/aws/invoke.test.js', () => {
+describe.only('test/unit/lib/plugins/aws/invoke.test.js', () => {
```

Comment:
Let's not commit this change (ideally it should be picked by linter)

Reply:
ouch, good catch